overrides/groovy/postInit/Post-Initial/Main/Mod-Specific/jei.groovy

```diff
@@ -1,10 +1,92 @@
 import com.nomiceu.nomilabs.util.LabsModeHelper
 
 /* Item Removals */
+//Unobtainable Ores
+void Remove_Trona(){
+mods.hei.removeAndHide(item('gregtech:ore_trona_0', 0)) //Trona Ore
+mods.hei.removeAndHide(item('gregtech:ore_trona_0', 1)) //Trona Ore
+mods.hei.removeAndHide(item('gregtech:ore_trona_0', 2)) //Trona Ore
+mods.hei.removeAndHide(item('gregtech:meta_dust_small:2031')) // Small Dust
+mods.hei.removeAndHide(item('gregtech:meta_crushed_purified:2031')) // Purified Crushed Ore
+mods.hei.removeAndHide(item('gregtech:meta_crushed:2031;')) // Crushed Ore
+mods.hei.removeAndHide(item('gregtech:meta_dust_tiny:2031')) // Tiny Dust
+mods.hei.removeAndHide(item('gregtech:meta_crushed_centrifuged:2031')) // Centrifuged Ore
+mods.hei.removeAndHide(item('gregtech:meta_dust_impure:2031')) // Impure Dust
+mods.hei.removeAndHide(item('gregtech:meta_dust_pure:2031')) // Pure Dust
+mods.hei.removeAndHide(item('gregtech:meta_dust:2031')) // Dust
+mods.hei.removeAndHide(item('gregtech:meta_crushed:2031')) // Crushed Ore
+}
+
+void Remove_Darmstadite(){
+if (LabsModeHelper.normal) {
```

Comment:
Note that here, the if statement (and its closing bracket) needs to have one tab of indentation, and the function calls inside the if statement needs to have two tabs of indentation.